gulpfile.js

```diff
@@ -115,6 +115,7 @@ gulp.task('cleanDefs', false, () =>
     .pipe(replace(/import.*$/gm, ''))
     .pipe(replace(/export =.+;$/gm, ''))
     .pipe(replace(/export default.+;$/gm, ''))
+    .pipe(replace(/: React.MouseEvent<[A-Za-z]+>/gm, ''))
```

Comment:
Actually it would cause a problem in some of our repos (v2) but the regexp is already at line 114 so you can remove it (I had the same problem)!